tests/unit/forklift/test_legacy.py

```diff
@@ -2020,6 +2027,7 @@ def test_upload_fails_with_wrong_filename(
             IFileStorage: storage_service,
             IMetricsService: metrics,
         }.get(svc)
+        monkeypatch.setattr(legacy, "_is_valid_dist_file", lambda *a, **kw: True)
```

Comment:
Filename validation is now happening after dist file validation, so we need to monkeypatch this.